Releases.md

```diff
@@ -6,6 +6,38 @@ https://github.com/denoland/deno/releases
 We also have one-line install commands at
 https://github.com/denoland/deno_install
 
+### 1.1.2 / 2020.06.26
+
+Changes in the CLI:
+
+- feat(unstable): Add Deno.fdatasyncSync and fdatasync (#6403)
+- feat(unstable): Add Deno.fstatSync and fstat (#6425)
+- feat(unstable): Add Deno.fsyncSync and fsync (#6411)
+- feat(unstable): Add Deno.ftruncate and ftruncateSync (#6243)
+- feat(unstable): Remove Deno.dir (#6385)
```

Comment:
It might be worth having unstable features/changes in a separate section just to avoid the predictable complains about us not following semver

Reply:
Or possibly don't mark these as feat?

Reply:
Added separate section and didn't mark as feats.